src/main/kotlin/org/rust/ide/settings/RsCodeInsightSettings.kt

```diff
@@ -28,5 +29,15 @@ class RsCodeInsightSettings : PersistentStateComponent<RsCodeInsightSettings> {
 
     companion object {
         fun getInstance(): RsCodeInsightSettings = service()
+
+        private val DEFAULT_EXCLUDED_PATHS: Array<ExcludedPath> = arrayOf(
+            ExcludedPath("std::borrow::Borrow", ExclusionType.Methods),
+            ExcludedPath("core::panicking"),
+            ExcludedPath("std::intrinsics::unreachable"),
+        )
```

Comment:
```suggestion
        private val DEFAULT_EXCLUDED_PATHS: Array<ExcludedPath> = arrayOf(
            // These imports interfere with `RefCell::borrow` & `RefCell::borrow_mut` and methods from
            // them are very rarely needed (mostly inside a `HashMap` implementations)
            // See https://github.com/intellij-rust/intellij-rust/issues/5805
            ExcludedPath("std::borrow::Borrow", ExclusionType.Methods),
            ExcludedPath("std::borrow::BorrowMut", ExclusionType.Methods),

            // Functions from this module are often suggested instead of `panic!()` macro, also
            // it is always unstable (with a stable alternative - `panic!()` macro)
            ExcludedPath("core::panicking"),

            // This method is often suggested in completion instead of `unreachable!()` macro, also 
            // it is always unstable (with a stable alternative - `core::hint::unreachable_unchecked`)
            ExcludedPath("std::intrinsics::unreachable"),
        )th("std::borrow::Borrow", ExclusionType.Methods),
            ExcludedPath("core::panicking"),
            ExcludedPath("std::intrinsics::unreachable"),
        )
```